gdc_client/upload/parser.py

```diff
@@ -85,7 +82,7 @@ def config(parser, upload_defaults):
     # TODO remove this and replace w/ top level host and port
     parser.add_argument('--server', '-s', type=str,
                         help='GDC API server address')
-    parser.add_argument('--http-chunk-size', '-c', type=int,
+    parser.add_argument('--upload-part-size', '-c', type=int,
```

Comment:
Leaving the `--http-chunk-size` and returning a message that says no longer supported, use `--upload-part-size` for same result might help users who already have anything that uses `--http-chunk-size` for upload

Reply:
It was `--part-size` before, but I see your point.

Reply:
After discussing with Ian, we decided to have `--part-size`, but mark it as deprecated and remove with the next release.